packages/controller/src/main.ts

```diff
@@ -2174,7 +2176,7 @@ async function processMessage(msg: ioBroker.SendableMessage): Promise<null | voi
 
                     if (changed) {
```

Comment:
Should we maybe also check running but blocked adapters and stop them? ... just as idea ...